examples/with-apollo-typescript/generated/index.tsx

```diff
@@ -0,0 +1,1341 @@
+import { gql } from '@apollo/client'
```

Comment:
If this file is automatically generated, it means we can remove it right?

Reply:
> If this file is automatically generated, do we need to include it in the PR?

Hm, in case we didn't include this after installing will be required to run command ```yarn generate```

Do you think will be better to exclude and add instructions to Readme ?

Reply:
I'm not sure. Let's for whatever `graphql-codegen` recommends. If the generated files have to be included then we include it for the example.

Reply:
> I'm not sure. Let's for whatever `graphql-codegen` recommends. If the generated files have to be included then we include it for the example.

There is nothing recommended regarding this file. In general way if you don't use codegen you will sotre all the queries inside the components or in another generic map, so it's the same with exception that codegen generates all the types and you just import them from generated file.

Reply:
My question is to whether it has to be ignored by `gitignore`. I would expect it to be generated once for development, and to be generated every time you make a build, to make sure it's always up to date. And because it's automatically generated and shouldn't be edited manually then it makes sense to not include. Unless `graphql-codegen` says otherwise.

Reply:
> My question is to whether it has to be ignored by `gitignore`. I would expect it to be generated once for development, and to be generated every time you make a build, to make sure it's always up to date. And because it's automatically generated and shouldn't be edited manually then it makes sense to not include. Unless `graphql-codegen` says otherwise.

I did it, thank you for review !